pkg/imexport/import.go

```diff
@@ -74,9 +74,14 @@ func Untardir(fs vfs.VFS, r io.Reader, dst string) error {
 				}
 			}
 
-			if _, err := io.Copy(file, tr); err != nil {
+			_, err = io.Copy(file, tr)
+			cerr := file.Close()
+			if err != nil {
 				return err
```

Comment:
Are we sure the file will be closed if Copy fail ? Otherwise we need to defer the file.Close().